src/packtPublishingFreeEbook.py

```diff
@@ -145,15 +145,19 @@ def grab_ebook(self, api_client):
         [user_data] = user_response.json().get('data')
         user_id = user_data.get('id')
 
+        product_response = api_client.get(PACKT_PRODUCT_SUMMARY_URL.format(product_id=product_id))
+        self.book_data = {'id': product_id, 'title': product_response.json()['title']}\
+            if product_response.status_code == 200 else None
+
+        if any(product_id == book['id'] for book in self.get_single_page_books_data(api_client, 0)):
+            logger.info('You have already claimed Packt Free Learning "{}" offer.'.format(self.book_data['title']))
```

Comment:
How about changing "offer" to "product" (I'd use "e-book" here but it doesn't have to be e-book)? But maybe the current message is also fine, I won't insist.

Reply:
how about just remove `offer`, imo `product` also weird.